### ISSUE TITLE
Import error : Cannot import name timezone

### ISSUE THREAD

**Issue by harshalmittal4** [chaoss/grimoirelab-manuscripts#126]
While running the command 
`manuscripts2 -n Perceval_Project -d PERCEVAL-REPORTS -s 2016-05-01 -e 2018-04-10 -i quarter -u http://localhost:9200/ --data-sources git github_issues github_prs \
--indices perceval_git perceval_github_issues perceval_github_prs -l logo.png`
to generate the report, the error arises.

**Comment by valeriocos** [comment]
Thank you for reporting this issue @harshalmittal4 . Could you provide the commands to generate the following indeces: `perceval_git perceval_github_issues perceval_github_prs`? I'd like to reproduce the error in my machine with the same steps you did.

Thank you!

**Comment by harshalmittal4** [comment]
Yes, I used the commands given in the readme here : [https://github.com/chaoss/grimoirelab-manuscripts/tree/master/manuscripts2](https://github.com/chaoss/grimoirelab-manuscripts/tree/master/manuscripts2)

**Comment by harshalmittal4** [comment]
Hi, please let me know if I'm missing something!
